borg/lrucache.py

```diff
@@ -1,42 +1,41 @@
-class LRUCache(dict):
```

Comment:
I see you're commenting on the overall diff.  I should have pointed you to the commit message :).  https://github.com/sourcejedi/borg/commit/9ba7daa9c706a76cc3eaf80661540457f0d80c56